### PR TITLE
[XDP] enhancements for profile API and report relative location

### PR DESCRIPTION
**Pull request by vipangul**
#### Problem solved by the commit
- Profile report needed tile locations as relative
- Profile API enhancements - differentiate beats ( transcation number ) and Bytes.
- Profile API enhancements - Changes to configure destination tile before source tile.

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
CR-1200024

#### How problem was solved, alternative solutions (if any) and why they were rejected
code cleanup

#### Risks (if any) associated the changes in the commit
low

#### What has been tested and how, request additional testing if necessary
Verified on VEK280

### CODE REVIEW

File: src/runtime_src/xdp/profile/database/static_info/aie_util.cpp [hunk 1]
```diff
@@ -331,6 +331,17 @@ namespace xdp::aie {
     return (absRow - rowOffset);
   }
 
+  /****************************************************************************
+   * Get string reprensation of relative row of given tile
```
Comment (pgschuey) [inline src/runtime_src/xdp/profile/database/static_info/aie_util.cpp:335]: please correct the spelling

File: src/runtime_src/xdp/profile/plugin/aie_profile/util/aie_profile_util.h [hunk 2]
```diff
@@ -44,6 +44,15 @@ namespace xdp::aie::profile {
     XAIE_MEM_MOD
   };
 
+  // Stream width in bits
+  const std::unordered_map<uint8_t, uint8_t> streamWidthMap = {
+    {XAIE_DEV_GEN_AIE,     32},
+    {XAIE_DEV_GEN_AIEML,   32}
+    // {XAIE_DEV_GEN_AIE2P,   64}
+    // {XAIE_DEV_GEN_AIE2PS,  64}
```
Comment (pgschuey) [inline src/runtime_src/xdp/profile/plugin/aie_profile/util/aie_profile_util.h:52]: Is there a reason why these are commented out?

**Review by pgschuey — APPROVED** [review]
Please address the minor items found

File: src/runtime_src/xdp/profile/plugin/aie_profile/util/aie_profile_util.h (hunk 2 above)
Comment (vipangul) [reply src/runtime_src/xdp/profile/plugin/aie_profile/util/aie_profile_util.h:52]: These definitions are yet not available in xaiegbl_defs.h , I will have to follow up with driver team to have it populated.
Currently it is used by AIE1/AIE2

**Review by jvillarre — APPROVED** [review]

**Pull request merged**